Tribler/Test/Core/test_torrent_utils.py

```diff
@@ -0,0 +1,31 @@
+import os
+from Tribler.Core.Utilities.torrent_utils import create_torrent_file
+from Tribler.Test.Core.base_test import TriblerCoreTest
+
+
+class TriblerCoreTestTorrentUtils(TriblerCoreTest):
+
+    FILE_DIR = os.path.abspath(os.path.dirname(os.path.realpath(__file__)))
+    TORRENT_DATA_DIR = os.path.abspath(os.path.join(FILE_DIR, u"data/torrent_creation_files/"))
+
+    def get_params(self):
+        return { "piece length" : 42, "comment" : "Proudly created by Tribler", "created by" : "someone",
+                 "announce" : "http://tracker.com/announce", "announce-list" : ["http://tracker.com/announce"],
+                 "httpseeds" : "http://seed.com", "urllist" : "http://urlseed.com/seed.php" }
+
+    def test_create_torrent_one_file(self):
+        create_torrent_file([self.TORRENT_DATA_DIR + "/file1.txt"], self.get_params(), self.created_torrent_one_file)
+
+    def created_torrent_one_file(self, result):
+        assert isinstance(result, dict)
+        assert result["base_path"] == self.TORRENT_DATA_DIR
+        assert result["success"]
+        assert os.path.isfile(result["torrent_file_path"])
+
+        os.remove(result["torrent_file_path"])
+
+    def test_create_torrent_two_files(self):
+        create_torrent_file([self.TORRENT_DATA_DIR + "/file1.txt", self.TORRENT_DATA_DIR + "/file2.txt"],
```

Comment:
you may want to also move `file1.txt` and `file2.txt` to an instance variable. `file1.txt` is now used at two places, changing only one of them (if you decide to) would break your test :)